src/test/jtx/AMM.h

```diff
@@ -329,8 +317,19 @@ class AMM
         std::optional<std::pair<Issue, Issue>> const& assets = std::nullopt,
         std::optional<ter> const& ter = std::nullopt);
 
-    void
-    bid(BidArg const& arg);
```

Comment:
I'd like to keep this function. Can just add `bidJson()` if needed for other tests.